core/src/main/java/com/infinum/jsonapix/core/common/JsonApiConstants.kt

```diff
@@ -57,6 +57,7 @@ object JsonApiConstants {
         const val FIND_TYPE = "findType"
         const val PARSE_TO_JSON_ELEMENT = "parseToJsonElement"
         const val JSONX_SERIALIZER_MODULE = "jsonApiXSerializerModule"
+        const val AS_JSON_X_HTTP_EXCEPTION = "asJsonXApiException"
```

Comment:
Should we rename to `AS_JSON_X_EXCEPTION` as well?

Reply:
Actually the method name is wrong 😓 
It should be `asJsonXHttpException()`. I'll rename it